scripts/explain.js

```diff
@@ -0,0 +1,137 @@
+// Note:
+//   The format of the spreadsheet should be:
+//
+//   | Explain | Definition                            | PM          | Contact | Link                                                                                              |
+//   | MAAS    | MAAS is a fast provisioning tool..... | Anton Smith | ~MAAS   | https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests |
+//
+// Authors:
+//   tbille, amylily1011
+
+var SPREADSHEET_ID = process.env.HUBOT_SPREADSHEET_ID;
+if (!SPREADSHEET_ID) {
+    console.log("Missing HUBOT_SPREADSHEET_ID in environment");
+}
+
+var CLIENT_EMAIL = process.env.HUBOT_SPREADSHEET_CLIENT_EMAIL;
+if (!CLIENT_EMAIL) {
+    console.log("Missing HUBOT_SPREADSHEET_CLIENT_EMAIL in environment");
+}
+
+var PRIVATE_KEY = process.env.HUBOT_SPREADSHEET_PRIVATE_KEY;
+if (!PRIVATE_KEY) {
+    console.log('Missing HUBOT_SPREADSHEET_PRIVATE_KEY in environment');
+} else {
+    PRIVATE_KEY = PRIVATE_KEY.replace(/\\n/g, '\n');
+}
+
+var CREDS = {
+    client_email: CLIENT_EMAIL,
+    private_key: PRIVATE_KEY
+};
+
+const { GoogleSpreadsheet } = require('google-spreadsheet');
+const doc = new GoogleSpreadsheet(SPREADSHEET_ID);
+
+var HTTPS_PROXY = process.env.HTTPS_PROXY;
+if (HTTPS_PROXY) {
+    doc.axios.defaults.proxy = false;
+    const HttpsProxyAgent = require('https-proxy-agent');
+    doc.axios.defaults.httpsAgent = new HttpsProxyAgent(HTTPS_PROXY);
+}
+
+var MATTERMOST_TOKEN_CMD_ACRONYM = process.env.MATTERMOST_TOKEN_CMD_ACRONYM;
+if (!MATTERMOST_TOKEN_CMD_ACRONYM) {
+    console.log("Missing MATTERMOST_TOKEN_CMD_ACRONYM in environment");
+}
+
+async function googleSpreadsheetHandler(explain) {
+    var explain;
+    await doc.useServiceAccountAuth(CREDS);
+    await doc.loadInfo();
+    const sheet = doc.sheetsByTitle['Explain'];
+    const sheet_why = doc.sheetsByTitle['Why'];
+    var rows = await sheet.getRows();
+
+    //get 'why' rows from sheet['Why']
+    var why_rows= await sheet_why.getRows();
+    var why_rows_length = why_rows.length;
+    var random_node = parseInt(Math.floor(Math.random()*(why_rows_length+1)));
+    //find the random element in why_row
+    var why_text = why_rows.find(a => a.rowIndex == random_node);
+    var display_text = why_text.why;
+
+    var responses = rows.filter(a => a.Explain && a.Explain.toUpperCase().trim() === explain);
+
+    var text = "";
+    responses.forEach(function (response) {
+       //if (text) { text = text; }
+        var link = response.Link ? response.Link : "";
+        var definition = response.Definition ? response.Definition : "";
+        var MM_Channel = response.Contact ? response.Contact: "";
+        var PM = response.PM ? response.PM: "";
+        var team = response.Team ? response.Team: "";
+        text = '\n'+ text + '|' + response.Explain + '| ' + definition + '| \n' + '| PM | ' + PM + '| \n' + '| Team | ' + team + '| \n' + '| Contact channel | ' + MM_Channel + ' | \n'+ '| Read more | '+ link + '|';
```

Comment:
It would be clearer to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for this. For example:
``` text = `\n ${text} | ${response.Explain} ... ` ```